templates/ortools/README.md

```diff
@@ -23,15 +23,15 @@ The most important files created are `main.py` and `input.json`.
 Run the command below to check that everything works as expected:
 
 ```bash
-python3 main.py -input input.json -output output.json -duration 30
+python3 main.py -input input.json -duration 30
```

Comment:
In our SDK templates we use the output file. I personally prefer this one that is writing directly to stdout, but should we harmonize the other templates as well? :thinking:

Reply:
The reason I changed this is because the line below reads:

> The optimal knapsack solution should be printed to stdout.

So we either:
- Save to `output.json` and delete the line below (probably best to be aligned with the other templates).
- Keep it as is to be aligned with what the README states.

Reply:
Oh yeah, that could have been me not updating the README when changing the line around all the time. :grimacing: 

Feel free to make a call. And good catch aligning this. I don't have a strong opinion whether writing to stdout or output.json is preferable.

Reply:
I wil change it to write to `output.json` per your suggestion.